k6/endpoints/related.js

```diff
@@ -0,0 +1,48 @@
+// run k6 run k6/endpoints/related.js
+// summary file uploaded at k6/summary/related.html
+import http from 'k6/http';
+import { sleep, check } from 'k6';
+import { Counter } from 'k6/metrics';
+import { htmlReport } from "https://raw.githubusercontent.com/benc-uk/k6-reporter/main/dist/bundle.js";
+
+// allows for incrementation of counter for every error
+const errors = new Counter('errors');
+
+// defines test run behaviors
+export const options = {
+  stages: [
+    { duration: '30s', target: 1500 }, // manually ramping the VUs to target of 2200
+    { duration: '30s', target: 2000 }
```

Comment:
Oh, I see that you tested a lot more VUs for your single endpoint tests, was it breaking in the batched test with more VUs?

Reply:
Sorta. The batched test reached the RPS of 1000 with less VUs because there was three requests per iteration